tests/src/string/tests/functions/from_raw_slice.rs

```diff
@@ -0,0 +1,13 @@
+use crate::string::tests::utils::{abi_calls::from_raw_slice, test_helpers::setup};
+
+mod success {
+
+    use super::*;
+
+    #[tokio::test]
+    async fn converts_from_raw_slice() {
+        let instance = setup().await;
+
+        from_raw_slice(&instance).await;
```

Comment:
I'd add some assertions here if possible

Reply:
Assertions are done in the contract as this is a heap type. This will be updated soon https://github.com/FuelLabs/sway-libs/issues/137